### PR TITLE
tests: migrate 20-cidr-limit to Ginkgo, deprecate Bash test

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

Relates-to: #1841 
Fixes: #2531

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

File: test/helpers/docker.go [hunk 1]
```diff
@@ -35,7 +35,9 @@ func (s *SSHMeta) ContainerCreate(name, image, net, options string) *CmdRes {
 // ContainerRm is a wrapper around `docker rm -f`. It forcibly removes the
 // Docker container of the provided name.
 func (s *SSHMeta) ContainerRm(name string) *CmdRes {
-	return s.ExecWithSudo(fmt.Sprintf("docker rm -f %s", name))
+	cmd := fmt.Sprintf("docker rm -f %s", name)
+	log.Debugf("removing container with command '%v", cmd)
```
Comment (eloycoto) [inline test/helpers/docker.go]: Miss the last `'` on `'%v`

File: test/runtime/Policies.go [hunk 2]
```diff
@@ -665,32 +668,58 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
 
+})
+
+var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
+	var once sync.Once
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"test": "RunPolicies"})
```
Comment (eloycoto) [inline test/runtime/Policies.go]: Here the name should be changed to `RunPolicyImportTests`

File: test/runtime/Policies.go [hunk 3]
```diff
@@ -713,7 +742,126 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
+		res := vm.ContainerCreate(helpers.Server, helpers.HttpdImage, helpers.CiliumDockerNetwork, "-l id.service2")
+
+		defer func() {
+			vm.ContainerRm(helpers.Server)
+		}()
+
+		res.ExpectSuccess()
+
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
```
Comment (eloycoto) [inline test/runtime/Policies.go]: assert message here.

File: test/runtime/Policies.go [hunk 4]
```diff
@@ -713,7 +742,126 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
...
+			cidrPolicy := fmt.Sprintf(`[{
+			"endpointSelector": {
+				"matchLabels": {"any:id.service2": ""}
+			},
+			"egress": [{
+				"toCIDR":
+					[ %s ]
+				}
+			]
+		}]`, cidrs)
```
Comment (eloycoto) [inline test/runtime/Policies.go]: Can you tab this please?

File: test/runtime/Policies.go [hunk 5]
```diff
@@ -713,7 +742,126 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 			}]
 		}]
 		}]`, ports)
-		testInvalidPolicy(script)
+		testInvalidPolicy(tooManyTCPPorts)
+	})
+
+	It("Test CIDR Limit", func() {
+		res := vm.ContainerCreate(helpers.Server, helpers.HttpdImage, helpers.CiliumDockerNetwork, "-l id.service2")
```
Comment (eloycoto) [inline test/runtime/Policies.go:750]: Why you create another container and why you don't use the sample containers? You're are creating the sample containers in the BeforeEach but not use that. I would like to use the same containers (httpd[1-3])

**Review by eloycoto — DISMISSED** [review]
A minor things but in overall LGTM

File: test/runtime/Policies.go (hunk 5 above)
Comment (eloycoto) [reply test/runtime/Policies.go:750]: Nothing here, I was in the wrong describe :-) Sorry!

Comment (ianvernon) [reply test/runtime/Policies.go:750]: No worries :)

**Comment by ianvernon** [comment]
test-me-please

File: test/runtime/Policies.go (hunk 4 above)
Comment (ianvernon) [reply test/runtime/Policies.go]: Fixed.

File: test/runtime/Policies.go (hunk 3 above)
Comment (ianvernon) [reply test/runtime/Policies.go]: Fixed.

File: test/runtime/Policies.go (hunk 2 above)
Comment (ianvernon) [reply test/runtime/Policies.go]: Fixed.

File: test/helpers/docker.go (hunk 1 above)
Comment (ianvernon) [reply test/helpers/docker.go]: Fixed.

**Review by eloycoto — APPROVED** [review]

**Pull request merged**